pkg/operator/configobserver/node/observe_latency_profile.go

```diff
@@ -0,0 +1,110 @@
+package node
+
+import (
+	"reflect"
+
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/klog/v2"
+
+	configv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/library-go/pkg/operator/configobserver"
+	"github.com/openshift/library-go/pkg/operator/events"
+)
+
+// LatencyConfigProfileTuple is used to set arg value pairs for each worker latency profile
+type LatencyConfigProfileTuple struct {
+	// path to required argument in observed config
+	// eg. extendedArguments.node-monitor-grace-period from observed config
+	// is []string{"extendedArguments", "node-monitor-grace-period"}
+	ConfigPath   []string
+	ProfileValue map[configv1.WorkerLatencyProfileType]string
```

Comment:
can we name this better please? It is not a single value

Reply:
Updated it to: `ConfigProfileValues map[configv1.WorkerLatencyProfileType]string` instead.

Reply:
to me ProfileConfigValues sounds better, although I guess there exist even a better name

Reply:
Let's keep it at `ProfileConfigValues` for now, I updated the struct!